phpBB/config/default/container/services_attachments.yml

```diff
@@ -0,0 +1,57 @@
+services:
```

Comment:
I'd rather move the filesystem service to its own file (like services_filesystem.yml) instead of duplicating stuff. You think it never causes issues until it does. Duplicating stuff is just a poor solution IMHO.